### PR TITLE
RDCC-2350: MFA - Add organisation_mfa_status table in PRD

### PR DESCRIPTION
**Pull request by ishratt**
### JIRA link (if applicable) ###

[RDCC-2350](https://tools.hmcts.net/jira/browse/RDCC-2350)

### Change description ###

Adding `organisation_mfa_status` table in PRD

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Organisation.java
Comment (nitishsingla1234) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Organisation.java:189]: Do we need an explicit getter and setter? Can we try with @Getter and @Setter annotations?

Comment (ishratt) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/domain/Organisation.java:189]: The @getter and @setter annotations are already present in the Organisation class, however all the fields have explicit getters, so to stay consistent I've kept the explicit getter. But obviously it can be removed.

File: src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java
Comment (SujeethKumarAri) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java:143]: I think default value is already been set in the db. is it required to set again?

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java
Comment (ishratt) [reply src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java]: Yes, you're right this is not required. I've just made a change and defaulted email in the entity class instead. Please kindly review the change @SujeethKumarAri

**Review by SujeethKumarAri — COMMENTED** [review]
Changes looks good to me. But I have not worked on PRD to know about existing functionalities. It would be better to get approval from someone who has already worked on PRD. Thanks

**Review by shreedhar-hmcts — APPROVED** [review]

**Review by SujeethKumarAri — APPROVED** [review]

**Review by AdilOoze — APPROVED** [review]

**Review by pksaxena01 — CHANGES_REQUESTED** [review]
Blocking to avoid accidental merge to master. We need to review and get approval.

**Review by nitishsingla1234 — CHANGES_REQUESTED** [review]
Please do not merge.

**Pull request closed without merging**
